search-app/src/cmr/search/system.clj

```diff
@@ -55,7 +79,11 @@
   "App logging level"
   {:default "info"})
 
-(def search-public-conf
+(def public-conf
```

Comment:
Let's go with the function, just to be safe -- I'll update them all, since this one might not have been getting overridden ...